build/build-buildimage-platforms.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+#
+# This script can be used to build the base platforms locally, before building the build image.
+# In Azure DevOps, this is taken care of by the pipelines defined in vsts/pipelines/buildimage-platforms.
+#
+
+set -e
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
+declare -r ACR_NAME='oryxdevmcr.azurecr.io'
+
+# Build Python
+docker build -f $REPO_DIR/images/build/python/prereqs/Dockerfile -t "$ACR_NAME/python-build-prereqs" $REPO_DIR
```

Comment:
Similar to how we do in build and runtime image build scripts, you an output the name of the built image into a file in the `artifacts` folder and then this file could be used as an input to the Docker task in the vsts pipeline to push those list of images. This approach was chosen as its analogous to how typically a .NET application would go through: build generates output in artifacts folder -> output is then published to some share or a web app.

Reply:
right, but this script isn't even used in DevOps - only for local development